Description: Use ‘sensible-browser’ to preview S5 slides.
 Use ‘sensible-browser’ (rather than ‘firefox’, which does not exist in Debian)
 as a program to preview S5 slides.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-08-21

--- a/tools/editors/emacs/rst.el
+++ b/tools/editors/emacs/rst.el
@@ -4032,7 +4032,7 @@
     ))
 
 ;; FIXME: Should be `defcustom` or use something like `browse-url`.
-(defvar rst-slides-program "firefox"
+(defvar rst-slides-program "sensible-browser"
   "Program used to preview S5 slides.")
 
 (defun rst-compile-slides-preview ()
